### DISCUSSION TITLE
how to install and use antoher icon pack ?

### DISCUSSION THREAD

**Discussion by jigujigu** [filamentphp/filament#3928] [Help] [answered]
Hi All,

How to install and use another icon pack (example icon from https://blade-ui-kit.com)
So I can use it on  $navigationIcon 

Thank You.

**Comment by danharrin** [comment, marked as answer]
https://github.com/owenvoke/blade-fontawesome

`$navigationIcon = 'fas-cloud'`

You can use the name of any Blade component in your app. These packages just add more Blade components
